### PR TITLE
Add SR2020 post competition email

### PR DESCRIPTION
**Pull request by sedders123**
The SR2019 post competition email was used as a template.

### CODE REVIEW

File: SR2020/2020-07-31-competition-wrap-up.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## Rewatch the streams
+
+If you'd like to relive the highs and lows of the competition livestreams of the matches remain available on YouTube, as well as a cut-down video of the spectacular final and tie breaker:
+
+- League Session 1: https://www.youtube.com/watch?v=xLL7SoQywf4
+- League Session 2: https://www.youtube.com/watch?v=7JoW4zXSeZE
+- League Session 3: https://www.youtube.com/watch?v=symUE1E4niI
+- League Session 4: https://www.youtube.com/watch?v=Y4h5P47j8jM
+- Knockouts and Final: https://www.youtube.com/watch?v=xBPVqsb_Ydk
+- Final and Tiebreaker: https://www.youtube.com/watch?v=xBPVqsb_Ydk
```
Comment (antoinepetty) [inline SR2020/2020-07-31-competition-wrap-up.md:31]:
```suggestion
- Final and Tiebreaker: https://www.youtube.com/watch?v=MAXpgODIqiI
```

File: SR2020/2020-07-31-competition-wrap-up.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+to: Student Robotics 2020
+subject: That's a wrap!
+---
+
+Hi all,
+
+First off, we'd like to thank you for taking part in Student Robotics 2020 and making it such a successful event! We were blown away by how well the teams adapted to the virtual competition in these uncertain times. An article about the competition, results, and game has been [posted on our website](https://studentrobotics.org/news/2020-07-25-post-competition/).
+
+Congratulation again to our winners:
```
Comment (antoinepetty) [inline SR2020/2020-07-31-competition-wrap-up.md:10]:
```suggestion
Congratulations again to our winners:
```

File: SR2020/2020-07-31-competition-wrap-up.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+| 2nd Place            | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/)                                                                    |
+| 3rd Place            | [LSS: Lawrence Sheriff School](http://www.lawrencesheriffschool.net/)                                                    |
+| Committee Award      | [SWI: South Wilts Grammar School](https://www.swgs.wilts.sch.uk/)                                                        |
+| Rookie Award         | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/)                                                                    |
+| First Robot Movement | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/) ([watch their robot](https://www.youtube.com/watch?v=KuFwqVyn_YA)) |
+| Online Presence      | [MAI: Gymnasium Markt Indersdorf](https://mai-robotics.de/)                                                              |
+
+## Rewatch the streams
+
+If you'd like to relive the highs and lows of the competition livestreams of the matches remain available on YouTube, as well as a cut-down video of the spectacular final and tie breaker:
```
Comment (antoinepetty) [inline SR2020/2020-07-31-competition-wrap-up.md:24]:
```suggestion
If you'd like to relive the highs and lows of the competition, livestreams of the matches remain available on YouTube, as well as a cut-down video of the spectacular final and tie breaker:
```

File: SR2020/2020-07-31-competition-wrap-up.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+- League Session 3: https://www.youtube.com/watch?v=symUE1E4niI
+- League Session 4: https://www.youtube.com/watch?v=Y4h5P47j8jM
+- Knockouts and Final: https://www.youtube.com/watch?v=xBPVqsb_Ydk
+- Final and Tiebreaker: https://www.youtube.com/watch?v=MAXpgODIqiI
+
+## Feedback
+
+We'd love to get your feedback on how the competition went for you this year, so we know what we did well, and where we can improve. Please complete the below short, anonymous survey. We also invite you to share this feedback form with your team members, and anyone else who attended the event.
+
+[Send us feedback](LINK_GOES_HERE)
```
Comment (RealOrangeOne) [inline SR2020/2020-07-31-competition-wrap-up.md:37]:
```suggestion
[Send us feedback](https://forms.gle/AwV2rPF18zqVtsUZ8)
```

**Comment by RealOrangeOne** [comment]
Sent to the teams who competed in the virtual competition

**Pull request merged**